awsconfigs/common/pipeline/multi-user/cache-deployer-cert-manager/cache-cert.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: cert-manager.io/v1
```

Comment:
Is this coming from open source repo ?

Reply:
cert-manager is being deployed by the open source repo 

The `Certificate` yaml is constructed using the cert-manager documentation 
https://cert-manager.io/docs/tutorials/acme/dns-validation/